apps/demos/Demos/Chat/Customization/jQuery/index.js

```diff
@@ -0,0 +1,70 @@
+$(async () => {
+  const chat = $('#chat').dxChat({
+    height: 712,
+    items: messages,
+    user: currentUser,
+    dayHeaderFormat,
+    onMessageEntered({ component, message }) {
+      component.renderMessage(message);
+    },
+    messageTimestampFormat,
+  }).dxChat('instance');
+
+  $('#chat-disabled').dxCheckBox({
+    value: false,
+    text: 'Disable Chat',
+    onValueChanged(data) {
+      chat.option('disabled', data.value);
+    },
+  });
+
+  $('#show-day-headers').dxCheckBox({
+    value: true,
+    text: 'Show Day Headers',
+    onValueChanged(data) {
+      chat.option('showDayHeaders', data.value);
+    },
+  });
+
+  $('#show-message-timestamp').dxCheckBox({
+    value: true,
+    text: 'Show Message Timestamp',
+    onValueChanged(data) {
+      chat.option('showMessageTimestamp', data.value);
+    },
+  });
+
+  $('#show-avatar').dxCheckBox({
+    value: true,
+    text: 'Show Avatar',
+    onValueChanged(data) {
+      chat.option('showAvatar', data.value);
+    },
+  });
+
+  $('#show-user-name').dxCheckBox({
+    value: true,
+    text: 'Show User Name',
+    onValueChanged(data) {
+      chat.option('showUserName', data.value);
+    },
+  });
+
+  $('#day-headers-format').dxSelectBox({
+    items: [dayHeaderFormat, 'dd.MM.yyyy', 'MMMM dd, yyyy', 'EEEE, MMMM dd'],
```

Comment:
kindly move the whole array to the `data.js`.
SelectBox's and Chart's initial value should be set to `dayHeaderFormats[0]`

Same for the messageTimestampFormat

Reply:
[a788963](https://github.com/DevExpress/DevExtreme/pull/28441/commits/a7889633ac3296ccaa25a0589dccff72c053b179)